config/release/kubemacpool.yaml

```diff
@@ -5,13 +5,15 @@ metadata:
     control-plane: mac-controller-manager
   name: kubemacpool-system
 ---
-apiVersion: admissionregistration.k8s.io/v1beta1
+apiVersion: admissionregistration.k8s.io/v1
 kind: MutatingWebhookConfiguration
 metadata:
   labels: null
   name: kubemacpool-mutator
 webhooks:
-- clientConfig:
+- admissionReviewVersions:
```

Comment:
```
AdmissionReviewVersions is an ordered list of preferred AdmissionReview versions the Webhook expects. API server will try to use first version in the list which it supports. If none of the versions specified in this list supported by API server, validation will fail for this object. If a persisted webhook configuration specifies allowed versions and does not include any versions known to the API Server, calls to the webhook will fail and be subject to the failure policy.
```

If I understand this correctly, this needs to hold the **pod**'s supported version(since this webhook checks pods), right?